packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -139,6 +149,13 @@ export function getMomentTemplateBanner(
                                 />
                             </div>
 
+                            {tickerSettings?.tickerData && (
+                                <MomentTemplateBannerTicker
+                                    tickerSettings={tickerSettings}
+                                    accentColour={'#d42d1a'}
```

Comment:
perhaps not important now, but could this be set in the `BannerTemplateSettings` that we pass in?